app/src/main/res/layout/sidewalk_lit.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same as above - id of `tactilePavingSidewalkImage` probably shouldn't be used for sidewalk_lit